libconnman-qt/networktechnology.cpp

```diff
@@ -81,15 +82,15 @@ TechnologyTracker::TechnologyTracker()
 
     // Monitor TechnologyAdded and TechnologyRemoved.
     QDBusConnection::systemBus().connect(
-                CONNMAN_SERVICE,
+                "net.connman",
```

Comment:
Seems like an unnecessary change.

Reply:
Using now the common again on this and later.